asyncio_run_in_process/_utils.py

```diff
@@ -87,3 +94,33 @@ def __reduce__(self) -> Any:
 def rebuild_exc(exc, tb):  # type: ignore
     exc.__cause__ = RemoteTraceback(tb)
     return exc
+
+
+def cleanup_tasks(task: 'asyncio.Future[Any]',
+                  *others: 'asyncio.Future[Any]',
```

Comment:
It'd be nice to have a docstring explaining how the different arguments are treated. I mean, it looks like they aren't treated the same, but I'm not really sure what's the difference

Reply:
Adding a docstring.

As for your confusion.  This *could* be written as `def cleanup_tasks(*tasks)`.  However, by doing it with the extra initial positional argument, it:

1. ensures that there is at least 1 task without having to validate `len(tasks) >= 1`
2. eliminates the need to pull off the first task via `tasks[0]`.